_docs/policies/tests.md

```diff
@@ -0,0 +1,47 @@
+# `status-go` Test Policy
+
+- [Creating Tests](#creating-tests)
+- [Flaky Tests](#flaky-tests)
+
+## Creating Tests
+
+- All new functionality MUST be introduced with tests that:
+  - Prove that the functionality performs as described
+  - Can be falsified
+  - Are resistant to fuzzing
+- All new tests MUST BE validated via 1000 local tests. Ensuring that the test runs and passes consistently every time gives confidence that the test is not flaky.
+    - add the `-count` flag to the test command eg: `-count 1000`
```

Comment:
The name of the param changed to `test.count`
```suggestion
    - add the `-test.count` flag to the test command eg: `-test.count 1000`
```

Reply:
How/where it changed? :thinking:
```
$ go help testflag
        -count n                                                                               
            Run each test, benchmark, and fuzz seed n times (default 1).
            If -cpu is set, run n times for each GOMAXPROCS value.                             
            Examples are always run once. -count does not apply to                             
            fuzz tests matched by -fuzz.
```

Reply:
I wasn't aware of this change, I checked the docs and I found one reference https://pkg.go.dev/testing#T.Parallel

> Parallel signals that this test is to be run in parallel with (and only with) other parallel tests. When a test is run multiple times due to use of -test.count or -test.cpu, multiple instances of a single test never run in parallel with each other.

But checking the latest version flag docs https://pkg.go.dev/cmd/go#hdr-Testing_flags I can see no reference to `-test.*`. 

It seems that `-test.*` just refers to the context of the flag rather than the actual usage of the flag